### ISSUE TITLE
Getting error when trying to use this package in a Lambda function

### ISSUE THREAD

**Issue by tpeterson39** [jkehler/awslambda-psycopg2#15]
I tried to use this package inside of my lambda function and I am still getting the following error:

"Unable to import module 'redshift_loader': No module named psycopg2"

I also tried building this from scratch per your instructions, but I am unsure of which directory is the newly created package. Some guidance would really be appreciated. 

Thank you,

Tyler

**Comment by tpeterson39** [comment]
Oaky, after following the instructions again, I realized that I should copy the entire "build" directory into my lambda_file.zip package. But I am still getting the same error: "No module named psycopg2".

This really sucks because this is such a good module when dev'ing locally.

**Comment by jkehler** [comment]
It's probably an issue with your permissions. Please reference #3

**Comment by tpeterson39** [comment]
I updated the permissions on my file and it's still not working, I'll attach a copy of my zip so you can see what I'm working with. This is the error I'm seeing...

```
Unable to import module 'redshift_loader': /var/task/psycopg2/_psycopg.so: invalid ELF header
```

I really appreciate your help, and the time you take to do this. Thank you.
[redshift_loader.zip](https://github.com/jkehler/awslambda-psycopg2/files/800169/redshift_loader.zip)

**Comment by jkehler** [comment]
It sounds to me like you are building it from scratch instead of using the existing library. You're build process is not creating a proper Linux formatted file. So either you're trying to build this on Windows or on Mac which will not work as Lambda is a Linux environment. 

When I copy my pre-built module in this github repo directly into your project it works fine.

**Comment by tpeterson39** [comment]
I had tried both ways and I couldn't get it to work... When I used the pre-built module, I had copied it into the lib folder in my zip and it couldn't import the module. But I just tried again, and you are right, when I just copied it into the zip I sent you (from a linux EC2 instance) lambda could load the function. 

Now to track down errors related to the credstash and crytography modules. Haha.

Thank you so much for taking the time to help out and respond to my questions. I really do appreciate it.
